packages/leva/package.json

```diff
@@ -22,19 +22,20 @@
     "react-dom": ">=16.8.0"
```

Comment:
From what I know, using `^16.8.0 || ^17.0.1 || ^18.2.0` is "safer" (than just `>=16.8.0`) since it won't automatically "accepts" future major versions of React, that could potentially break things

In this form, you'll have to add `|| ^19.0.0` when the library has tested React@19 was ok with

Reply:
Fixed!